codecov_cli/services/commit/__init__.py

```diff
@@ -53,15 +53,21 @@ def send_commit_data(
     enterprise_url,
     args,
 ):
-    # this is how the CLI receives the username of the user to whom the fork belongs
-    # to and the branch name from the action
-    tokenless = os.environ.get("TOKENLESS")
-    if tokenless:
-        headers = None  # type: ignore
-        branch = tokenless  # type: ignore
-        logger.info("The PR is happening in a forked repo. Using tokenless upload.")
+    # Old versions of the GHA use this env var instead of the regular branch
+    # argument to provide an unprotected branch name
+    if tokenless := os.environ.get("TOKENLESS"):
+        branch = tokenless
```

Comment:
branch = os.environ.get("TOKENLESS", None) could have worked too

Reply:
that would clobber whatever was already in branch

Reply:
@matt-codecov ahhh that's a good point